ballerina/types.bal

```diff
@@ -24,6 +24,4 @@ public type Configuration record {
     ai:MetadataFilters filters?;
     # Optional sparse vector for hybrid search operations
     ai:SparseVector sparseVector?;
-    # Number of top similar vectors to return in queries
-    int topK = 5;
```

Comment:
Just to consider: This is a breaking change.

Reply:
Yes, as discussed with @shafreenAnfar, this decision was made intentionally
@Nuvindu let's bump the minor version.